<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditor</class>
 <widget class="QWidget" name="ItemEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка задачи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1">
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="textEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="textUrgency">
       <property name="text">
        <string>Срочность</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QDialogButtonBox" name="buttonConfirm">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="textSkill">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>  Развиваемый навык</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="textTaskname">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>     Содержание задачи:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="textDeadline">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QSlider" name="sliderUrgency">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="textFear">
       <property name="text">
        <string>Страх</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QSlider" name="sliderDifficulty">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="textDifficulty">
       <property name="text">
        <string>Сложность</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QSlider" name="sliderFear">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QPushButton" name="buttonDate">
       <property name="text">
        <string>ДД.ММ.ГГГГ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
